.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+# Define stages for the CI/CD pipeline
+stages:
+  - build_docker_image
+
+variables:
+  CONTAINER_IMAGE: ${CI_REGISTRY_IMAGE}:eard-${CI_COMMIT_SHORT_SHA}
```

Comment:
This will build a new image for every commit regardless of branch. It might make sense to limit it to main?

Reply:
Actually this will disrupt the backend repo if it is using :latest